README.md

```diff
@@ -1,10 +1,18 @@
-# Project Name
-Module 3 Maths Magician
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Maths Magician
 
 > In this project Math Magician built in React to do simple calculation application like multiplication division substraction and addition operations.
 
-It have one interface with three parts (numerical party, operator party and display part)
-> ![screenshoot](./Capture.PNG)
+It have one interface with three parts (numerical party, operator party and display part).
+
+> The project consisting of 3 pages: Home, Calculator, and Quote.
+
+> ![screenshoot](./Appscreenshoot.PNG)
+
+# Getting Started with Create React App
```

Comment:
- [ ] At this stage, the README should be informing us on how to get started with this particular App and not CRA. Generally, the README needs editing to match [this particular template](https://github.com/microverseinc/readme-template). Please ensure the README is updated to help make this repo look professional.